bin/oneoff/backfill_data/scholarship_info

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+Pd::Application::Teacher1920Application.find_each do |application|
+  if application.properties["scholarship_status"]
+    application.update_scholarship_status(application.properties["scholarship_status"])
```

Comment:
What's the plan to handle potential errors in this script?  Do we want to wrap the whole thing in a transaction, or rescue errors and output which records couldn't be updated?

Reply:
*facepalm* I'll add error handling in - I forgot running a script doesn't have the automatic feedback of running things in the console!

Reply:
might also want to remove the `scholarship_status` property here